sourcefiles/ctenums.py

```diff
@@ -982,8 +982,7 @@ def get_mt_woe_treasures(cls):
             cls.MT_WOE_KEY
         ]
 
-
-class EnemyID(StrIntEnum):
+class BossID(StrIntEnum):
```

Comment:
I do not support partitioning the `EnemyID` enumeration.  This enumeration represents the internal indexing of enemies as in the CT Rom.  If you change it, then every function which expects `EnemyID` has to be changed to some union type which then is subject to further changes is more subdivisions are made.  If you wish to partition enemies, write a function that returns the relevant subsets.

Reply:
I said very clearly in my original comment what I wanted.
>  If you wish to partition enemies, write a function that returns the relevant subsets.
I want a function that returns the relevant subset of EnemyID.  Whether it's a standalone function of method of EnemyID, I do not care.  I do not want a second, duplicate enumeration.

Reply:
Updated,  refactored into a class method under `EnemyID`